src/app/clusters/testing/ClusterTester.h

```diff
@@ -69,6 +69,34 @@ namespace Test {
 //     ASSERT_GT(it.GetValue().label.size(), 0u);
 // }
 //
+
+// Detection traits for Encode() and EncodeForWrite()
+template <typename...>
+using void_t = void;
```

Comment:
Should we not just include `<type_traits>` instead? https://en.cppreference.com/w/cpp/types/void_t.html